README.md

```diff
@@ -1 +1,48 @@
-# community
\ No newline at end of file
+# ROS 2 Security Working Group
+The Security Working Group's mission is to advocate for and implement security features within ROS 2. The working group negotiates a balance between the benefits of security and the enablement of technology, striving for a secure-by-default design which can be customized to suit a targeted security profile.
+
+This document outlines the governance of the ROS 2 Security Working Group. Updates to this document will be handled in the same manner as other project updates governed by the Working Group.
+
+## Subprojects
+A significant portion of the Security Working Group's mission is achieved by maintaining ROS projects. The following projects are owned by this Working Group:
+* _TBD:  list of project names_
+
+### Adding subprojects
+To request that the Security Working Group take on ownership and maintainership of a particular project, create a new issue in this repository using the appropriate issue template. You may be requested to present your proposal at the next Security Working Group meeting. Approvers will vote on whether or not the Working Group will accept the project.
+
+### Standards for subprojects
+Subprojects must meet the following criteria:
+* Builds must pass against ROS 2 master
+* Test suite must pass
+* Test coverage must be greater than 50%
+* The ROS 2 standard linter set must be enabled and adhered to
+* Builds must have 0 warnings
+
+## Governance
+These guidelines are designed exclusively to help the Working Group achieve its mission, but are subject to change should they become cumbersome or ineffective.
+
+### Meetings
+* The working group typically meets on the last Tuesday of each month, at alternating times to accomodate our community's varied timezones
+* Meetings are announced and an agenda created on the ROS Discourse using the [wg-security tag](https://discourse.ros.org/tags/wg-security)
+* Meeting notes are kept on the [ROS Wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security)
+* Meetings are recorded and available [on YouTube](https://www.youtube.com/playlist?list=PLpUh4ScdBhSMaEekJ8xeAAGmWUgR9S1K_)
+
+### Communication channel
+The Security Working Group has a few communication channels:
+* ROS Discourse using the [wg-security tag](https://discourse.ros.org/tags/wg-security)
+* Chat using a [Matrix room](https://matrix.to/#/!LcRLnAIRWjSCfZmMeD:matrix.org?via=matrix.org)
+
+### Roles
+Security Working Group members may act in one or more of the following roles:
+* __Member__
+  * Attend a majority of the Security Working Group meetings
+  * Responsible for triaging issues
+* __Reviewer__
+  * All reviewers are members
+  * Responsible for reviewing pull requests
```

Comment:
It appears that both members and reviewers (actually anyone) can comment and review a PR on Github.
So my understanding is that reviewers have a responsibility / obligation to review PRs while member do not *have* to do it.

What would the workflow be in this context?
Users open PR
 - member triages it and assigns a reviewer 
   - reviewer iterates with contributor, once satisfied it assigns an approver to the PR
        - the approver iterates with contributor and once satisfied merges the PR

Reply:
You're right, anyone can make or review a PR, or triage an issue, this is about responsibilities in the team more than about capabilities. If you're a member, you're responsible for helping to triage. If you're a reviewer, you're responsible for reviewing. If you're an approver, you're responsible for agreeing for the WG to take on new responsibilities (whether that's by approving a PR or by accepting new projects). These build on each other, of course-- approvers should also be working to triage, for example.

The workflow can be what you outlined, but it could be as simple as:

- PR is opened
  - Approver reviews and iterates until satisfied, and approves

I recommend some sanity checks, like ensuring there are two approvals before merging, but that's a little out of scope here.